src/Arcus.WebApi.Integration/Jobs/TempAzureKeyVaultSecret.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Threading.Tasks;
+using GuardNet;
+using Microsoft.Azure.KeyVault;
+
+namespace Arcus.WebApi.Integration.Jobs 
+{
+    /// <summary>
+    /// Representation of a Azure Key Vault secret with a lifetime the same as the type (dispose type = delete secret).
+    /// </summary>
+    public class TempAzureKeyVaultSecret : IAsyncDisposable
```

Comment:
Let's call it `TemporaryAzureKeyVaultSecret` then, I'm not really a fan of abbreviations personally.

Reply:
Done.

Reply:
Thanks!